File: mcovid_shield/lib/model/statistics.dart
import 'dart:convert';

class Statistics {
  final int confirmed;
  final int todayConfirmed;
  final int recovered;
  final int todayRecovered;
  final int deaths;
  final int todayDeaths;
  final bool casesIcon;
  final bool recoversIcon;
  final bool deathsIcon;
  final int tests;

  Statistics({this.confirmed, this.recovered, this.deaths, this.todayConfirmed, this.todayRecovered, this.todayDeaths, this.casesIcon, this.recoversIcon, this.deathsIcon, this.tests});

  factory Statistics.fromJson(Map<String, dynamic> json) {
    return Statistics(
      confirmed: json['cases'],
      recovered: json['recovered'],
      deaths: json['deaths'],
      //get today data using todays and yesterday cases!
      todayConfirmed: json['todayCases'],
      todayRecovered: json['TodayRecovers'],
      todayDeaths: json['todayDeaths'],
      //get icons data
      casesIcon: json['casesIcon'],
      recoversIcon: json['recoversIcon'],
      deathsIcon: json['deathsIcon'],
      //Tests
      tests: json['tests'],
    );
  }
}

File: Server/statistics.php
<?php
// fetch data
$today = json_decode(file_get_contents('https://corona.lmao.ninja/v2/countries/morocco'));
$yesterday = json_decode(file_get_contents('https://corona.lmao.ninja/v2/countries/morocco?yesterday=true'));

// append today recovers info
$today->TodayRecovers = ($today->recovered - $yesterday->recovered);

// cases icons data
$today->casesIcon = ($today->todayCases > $yesterday->todayCases) ? true : false;

//need corrections!!
$today->recoversIcon = ($today->recovered > $yesterday->recovered) ? true : false;

// death icons data
$today->deathsIcon = ($today->todayDeaths > $yesterday->todayDeaths) ? true : false;

// show new object
echo json_encode($today);

